startup/GafferScene/renderPassOptions.py

```diff
@@ -40,3 +40,7 @@
 Gaffer.Metadata.registerValue( "option:renderPass:enabled", "label", "Enabled" )
 Gaffer.Metadata.registerValue( "option:renderPass:enabled", "description", "Whether the render pass is enabled for rendering." )
 Gaffer.Metadata.registerValue( "option:renderPass:enabled", "defaultValue", IECore.BoolData( True ) )
+
+Gaffer.Metadata.registerValue( "option:renderPass:type", "label", "Type" )
+Gaffer.Metadata.registerValue( "option:renderPass:type", "description", "The type of the render pass." )
```

Comment:
Can we provide a little more information here? Maybe something like this?

> This provides simple setup for renders such as reflection and shadow passes, typically by assigning custom shaders to the objects specified by `Casters` and `Catchers`. Use a RenderPassShaders node to customise the shaders used for this purpose.
>
> > Hint : Render pass types and their behaviours can be customised using the RenderPassTypeAdaptor API.

Reply:
Updated in a16fc3a94e6bd4198b4c709ff7f137b34fc46158.